web/styles/zulip.css

```diff
@@ -2218,16 +2221,23 @@ body:not(.spectator-view) {
             top: 0.05em;
         }
 
-        .dropdown-list-delete,
-        .dropdown-list-edit {
-            position: absolute;
-            top: 0;
-            right: 5px;
+        .dropdown-list-buttons {
+            display: flex;
             visibility: hidden;
         }
 
-        .dropdown-list-edit {
-            right: 30px;
+        .dropdown-list-item-name {
+            display: grid;
+            grid-template-columns: 1fr auto;
+            align-items: center;
+        }
+
+        .dropdown-list-item-description {
+            grid-column: 1;
```

Comment:
And yes, this is what makes placement even more brittle, when we have to resort to referencing specific columns and rows. Something like `grid-area: item-description` is more robust and easier to reason about.